engine/src/flutter/shell/platform/embedder/fixtures/main.dart

```diff
@@ -1632,3 +1632,74 @@ void testSendViewFocusChangeRequest() {
     direction: ViewFocusDirection.backward,
   );
 }
+
+@pragma('vm:entry-point')
+// ignore: non_constant_identifier_names
+Future<void> a11y_main_multi_view() async {
+  // 1: Return initial state (semantics disabled).
+  notifySemanticsEnabled(PlatformDispatcher.instance.semanticsEnabled);
+
+  // 2: Add the first view (implicitly handled by PlatformDispatcher)
+  // 3: Add the second view (implicitly handled by PlatformDispatcher)
+
+  // 4: Await semantics enabled from embedder.
+  await semanticsChanged;
+  notifySemanticsEnabled(PlatformDispatcher.instance.semanticsEnabled);
+
+  // 5: Return initial state of accessibility features.
+  notifyAccessibilityFeatures(PlatformDispatcher.instance.accessibilityFeatures.reduceMotion);
+
+  // 6: Await accessibility features changed from embedder.
+  await accessibilityFeaturesChanged;
+  notifyAccessibilityFeatures(PlatformDispatcher.instance.accessibilityFeatures.reduceMotion);
```

Comment:
FYI, I think you can skip this for the multi-view test to cut down the lines of code. Feel free to keep as-is though!